package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>iceoryx2_cxx</name>
+    <version>0.3.0</version>
+    <description>iceoryx2 with c/c++ bindings</description>
+    <maintainer email="iceoryx-oss-support@ekxide.io">ekxide developers</maintainer>
+    <license>Apache 2.0</license>
+    <url type="bugtracker">https://github.com/eclipse-iceoryx/iceoryx2/issues</url>
+    <url type="repository">https://github.com/eclipse-iceoryx/iceoryx2</url>
+
+    <buildtool_depend>cmake</buildtool_depend>
+    <depend>iceoryx_hoofs</depend>
```

Comment:
Doesn't this cause build issues?

Reply:
@elBoberido If the right version of `iceoryx_hoofs` is in the colcon workspace it works. 

Nevertheless, I have added a version attribute to the dependency which will print a warning if the version is too low. Doesn't fail the build outright though with this reasoning (prints other errors that are a result of the version mismatch along with the warning.

Reply:
Right, this will only be a problem once we become part of the ros2.repos list